mage_ai/frontend/components/datasets/FeatureProfiles.tsx

```diff
@@ -88,95 +106,56 @@ function FeatureProfile({
   const medianValue = statistics?.[`${uuid}/median`];
   const modeValue = statistics?.[`${uuid}/mode`];
   const numberOfInvalidValues = statistics?.[`${uuid}/invalid_value_count`];
+  const numberOfOutliers = statistics?.[`${uuid}/outlier_count`];
+  const skewness = statistics?.[`${uuid}/skew`];
+  const stddev = statistics?.[`${uuid}/std`];
 
-  let entries = [];
-  
-  if (COLUMN_TYPE_NUMBERICAL_WITH_DATETIME_LIKE.includes(columnType)) {
-    entries = [
-      [
-        'Type',
-        'Count',
-        'Unique',
-        'Missing',
-      ],
-      [
-        COLUMN_TYPE_HUMAN_READABLE_MAPPING[columnType] || columnType,
-        numberOfValues,
-        numberOfUniqueValues,
-        numberOfNullValues,
-      ],
-      [
-        columnType === ColumnTypeEnum.DATETIME ? 'Median' : 'Mean',
-        'Minimum',
-        'Maximum',
-        'Invalid',
-      ],
-      [
-        columnType === ColumnTypeEnum.DATETIME ? medianValue : roundNumber(meanValue, 3),
-        typeof minValue === 'string' ? minValue: roundNumber(minValue, 3),
-        typeof maxValue === 'string' ? maxValue: roundNumber(maxValue, 3),
-        numberOfInvalidValues,
-      ],
-    ];
-  } else {
-    entries = [
-      [
-        'Type',
-        'Count',
-        'Unique',
-      ],
-      [
-        COLUMN_TYPE_HUMAN_READABLE_MAPPING[columnType] || columnType,
-        numberOfValues,
-        numberOfUniqueValues,
-      ],
-      [
-        'Missing',
-        'Mode',
-        'Invalid',
-      ],
-      [
-        numberOfNullValues,
-        modeValue,
-        numberOfInvalidValues,
-      ],
-    ];
-  }
+  const entries = [
+    columnType,
+    numberOfUniqueValues,
+    meanValue,
+    medianValue,
+    modeValue,
+    minValue,
+    maxValue,
+    numberOfNullValues,
+    numberOfInvalidValues,
+    numberOfOutliers,
+    skewness,
+    stddev,
+  ];
 
   const featureSetId = featureSet.id;
   const featureUuid = feature?.uuid;
-  const featureId = getFeatureIdMapping(featureSet)[featureUuid]
+  const featureId = getFeatureIdMapping(featureSet)[featureUuid];
 
   return (
-    <FlexContainer>
-      <Flex flex={1}>
-        <CellStyle>
+    <Flex flex={1} flexDirection="column">
```

Comment:
Would `FlexContainer` behave the same as `Flex` here? Seems strange to use the `Flex` component instead of `FlexContainer`.

Reply:
in terms of having the appropriate semantic components

Reply:
you're probably right, I didn't even realize the difference until today. I can fix

Reply:
Just checked, the reason I used `<Flex>` is because this is a column subcomponent that eventually gets repeated multiple times inside of a `<FlexContainer>` table. I had to make this a `<Flex>` so that columns would grow vertically downwards.

Reply:
Ok that makes sense